asciinema2gif

```diff
@@ -5,18 +5,18 @@
 # Halt on error.
 set -e
 
-rundir=$PWD
+readonly rundir="$(basename "${0}")"
```

Comment:
it turns out:
`phantomjs render.js $1` would work
while 
`phantomjs render.js "${1}"` would not